README.md

```diff
@@ -118,7 +118,7 @@ Here is a list of Bluepill exit codes. If a Bluepill execution has multiple exit
 
 ## Requirements
 
-Bluepill only works with **Xcode 11.3**. If you're looking for old Xcode support, please check out the other branches:
+Bluepill officially supports **Xcode 11.4**. If you're looking for old Xcode support, please check out the below branches:
```

Comment:
Maybe "please checkout the following branches"?

Reply:
Sure, I will change it in my next PR.